### ISSUE TITLE
Reversed polarity on C5, C12

### ISSUE THREAD

**Issue by richardstefun** [anotherlin/olympic-white#1]
Hello,

I think that the schematic diagram has those two components with reversed, incorrect polarity.

<img width="684" alt="image" src="https://user-images.githubusercontent.com/52629010/200156436-cfbf3e19-75e5-4c7b-96bd-4e8ca7d67761.png">

**Comment by anotherlin** [comment]
Hi Richard,

First of all, thank you for your interest and taking your time to look at this project.
Yes, you're absolutely right, C5 and C12 have polarity reverse connected. This needs to be corrected.

The project has been frozen since the first check-in on github, and the layout is actually wrong.
The way pads are around the TPS65130, makes it very difficult (thermal issue), if not impossible to solder properly (I never succeeded).
Also, it makes use of 0603 resistors (or may be even 0402), which are horrendous for hand soldering.
Maybe just do everything with 0805 which is a good balance between small size and ease of use .
Using through-holes for the "programming" resistors is also a bad idea, the pads take a lot of space.
And why not just use 0805 for everything.

Most importantly, I should add formulas for computing the correct values of capacitors and inductors for various current requirements.
There is no "one size fits all" for that. Selecting the programming resistors, is not enough.

So basically, this project needs a reboot and still a lot of work to make it functional.
I'll try to work on it again soon.

Best regards,

**Comment by richardstefun** [comment]
Hi Lin,

I'm glad that someone found this IC useful too. I tried to design my board around it a while ago, but unfortunately I ran into some issues with resistor/inductor calculations, so I decided to look online for some open source designs based on this chip. That's how I found this project. 

Concerning the components size, I think that the 0603 size is generally quite comfortable to work with even with basic SMT tools. The IC should be soldered with solder paste, or let the folks at JLCPCB do it for you 😄

If the board should be, at least partially finished manually, we can add some table with most commonly used voltages (+- 9/12/15...V) and let the user choose the right C/R/L for his need. 

There are some existing boards that use this IC available on AliExpress (with no documentation), but I still think that rebooting this project would have some value for someone trying to extend it, or learn something new..
